src/charm.py

```diff
@@ -269,6 +270,12 @@ def kubernetes_context(self) -> Dict[str, Any]:
             "kubelet_dir": self.kubelet_dir,
         }
 
+    @cached_property
+    def _client(self) -> Client:
+        """Lightkube Client instance."""
+        client = Client(field_manager=f"{self.model.app.name}")
+        return client
+
```

Comment:
Ack, this charm is a subordinate of kubernetes-control-plane -- should get a client using `/root/.kube/config`